src/main/java/io/github/sefiraat/networks/slimefun/network/NetworkControlX.java

```diff
@@ -117,6 +133,7 @@ private void tryBreakBlock(@Nonnull BlockMenu blockMenu) {
         definition.getNode().getRoot().addItemStack(resultStack);
 
         if (resultStack.getAmount() == 0) {
+            this.blockCache.add(new BlockPosition(targetBlock));
```

Comment:
targetPosition

Reply:
Whoops lol, not sure why I didn't put that